src/cf-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-core",
-  "version": "4.0.1",
+  "version": "4.2.0",
```

Comment:
Ah heck. I really wonder if we need to do condensed Changelog entries right before opening a bot PR when there are multiple changes like this.